github/ci/jenkins-master/templates/04-jobs.groovy

```diff
@@ -0,0 +1,40 @@
+job('kubevirt-functional-tests') {
+    parameters {
+        stringParam('sha1', '', 'commit to build')
+    }
+    scm {
+        git {
+            remote {
+                github('{{ githubRepo }}')
+                refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+            }
+            branch('${sha1}')
+            extensions {
+                relativeTargetDirectory('go/src/kubevirt.io/kubevirt')
+            }
+        }
+    }
+    triggers {
+        githubPullRequest {
+            admins(['rmohr', 'fabiand', 'stu-gott', 'admiyo'])
+            cron('H/2 * * * *')
+            triggerPhrase('OK to test')
+            allowMembersOfWhitelistedOrgsAsAdmin()
+            extensions {
+                commitStatus {
+                    context('kubevirt-functional-tests/jenkins/pr')
+                    triggeredStatus('kubevirt-functional-tests/jenkins/pr')
+                    startedStatus('kubevirt-functional-tests/jenkins/pr')
+                    statusUrl('https://github.com/kubevirt/kubevirt')
+                    completedStatus('SUCCESS', 'All is well')
+                    completedStatus('FAILURE', 'Something went wrong. Investigate!')
+                    completedStatus('PENDING', 'still in progress...')
+                    completedStatus('ERROR', 'Something went really wrong. Investigate!')
+                }
+            }
+        }
+    }
+    steps {
+        shell('cd go/src/kubevirt.io/kubevirt && bash automation/test.sh')
```

Comment:
Sorry for the confusion. The official pull request is here: https://github.com/kubevirt/kubevirt/pull/339

So far I tested against a pull request in my own repo: https://github.com/rmohr/kubevirt/pull/3